### ISSUE TITLE
Two examples need renaming to match lib name

### ISSUE THREAD

**Issue by kattni** [adafruit/Adafruit_CircuitPython_ADS1x15#39]
Two examples begin with `ads1015_` and `ads1115_`. They should each begin with `ads1x15_ads1015_` and `ads1x15_ads1115_` to match the CIrcuitPython library standards and make it clearer in the example bundle which library they are associated with.

The lines including this file name in the `docs/examples.rst` file must also be updated to reflect the name change, e.g.:
```
.. literalinclude:: ../examples/current_file_name.py
    :caption: examples/current_file_name.py
    :linenos:
```
would be updated to:
```
.. literalinclude:: ../examples/current_file_name_changed.py
    :caption: examples/current_file_name_changed.py
    :linenos:
```

**Comment by razinc** [comment]
Hi @kattni. I would like to take this but I don't have a clear picture of the issue. One of the requirements is changing the word `ads1015_` to `ads1x15_ads1015_` & `ads1115_` to `ads1x15_ads1115_` in `docs/examples.rst`. Does it mean I have to put the prefix `ads1x15` to each file in this repo?

**Comment by kattni** [comment]
Hello, @razinc. 

The prefix `ads1x15_` needs to be added to two of the example files in this repo (the two files that include `simpletest` in the name), the two other examples already have that as the beginning of their name (the two files that do not include `simpletest` in their name). 

Once the filenames are changed, the `docs/examples.rst` file needs the file names in it updated, as it includes both of the files that need to be renamed.

**Comment by razinc** [comment]
Okay. Let me take this, please.

**Comment by dherrada** [comment]
Closed. Fixed in #41
